client/src/ErrorBoundary.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+class ErrorBoundary extends React.Component {
```

Comment:
we are not working with class components, change to functional, and use the react error bounderies library from npm